ichnaea/locate/provider.py

```diff
@@ -596,6 +600,17 @@ def should_locate(self, data, location):
     def locate(self, data):
         location = self.location_type(query_data=False)
 
+        if self.redis_client:
+            rate_key = 'fallback:{time}'.format(time=int(time.time()))
+            limit_reached = rate_limit(
+                self.redis_client,
+                rate_key,
+                maxreq=self.settings['ratelimit'],
```

Comment:
From the config file the setting comes in as a string. Either we need to cast it to an int here or provide a custom `__init__` for the fallback provider which does the parsing and assigns it as something like `self.ratelimit`. I'd prefer to put the parsing logic into an init.